Cartfile

```diff
@@ -1 +1 @@
-github "antitypical/Result"
+github "antitypical/Result" >= 0.3.2
```

Comment:
This should really be `~> 0.3.2`, since we won't be compatible with 0.4 and later. Sorry I wasn't clearer before.